versions.rst

```diff
@@ -32,6 +32,12 @@ Unsupported Versions
    :file: include/end-of-life.csv
 
 
+Python Release Cycle
+====================
+
+.. mermaid:: include/release-cycle.mmd
+
+
```

Comment:
I would put this above the two tables.

Reply:
Updated!

Shall we also ditch the subheading:

```rst
Python Release Cycle
====================
```

so it's directly under the intro "The main branch is currently..." paragraph?